### ISSUE TITLE
SendViewControllerTests failing due to localisation issues.

### ISSUE THREAD

**Issue by eviltofu** [AlphaWallet/alpha-wallet-ios#5944]
Tests appear to be failing due to localisation issues: `testNativeCryptocurrencyAllFundsValueSpanish(): XCTAssertEqual failed: ("Optional("10000,0000")") is not equal to ("Optional("10000")")`
```text
Failing tests:
	SendViewControllerTests.testERC20AllFunds()
	SendViewControllerTests.testERC20AllFundsEnglish()
	SendViewControllerTests.testERC20AllFundsSpanish()
	SendViewControllerTests.testERC20IntialAllFunds()
	SendViewControllerTests.testNativeCryptocurrencyAllFundsValueEnglish()
	SendViewControllerTests.testNativeCryptocurrencyAllFundsValueSpanish()
	SendViewControllerTests.testScanEip681QrCodeEnglish()
	SendViewControllerTests.testScanEip681QrCodeEnglish()
	SendViewControllerTests.testScanEip681QrCodeSpanish()
	SendViewControllerTests.testScanEip681QrCodeSpanish()
	SendViewControllerTests.testScanEip681QrCodeSpanish2()
```

**Comment by hboon** [comment]
Not sure if it's localization.

**Comment by hboon** [comment]
@oa-s would you help to check? Might be bad

**Comment by oa-s** [comment]
Closed via #5758
